### PR TITLE
Support multiple test packages

### PR DESCRIPTION
**Pull request by kokosing**
Support multiple test packages

### CODE REVIEW

File: tempto-runner/src/main/java/io/prestosql/tempto/runner/TemptoRunner.java [hunk 1]
```diff
@@ -120,8 +121,9 @@ private XmlSuite getXmlSuite()
         testSuite.setFileName("tempto-tests");
         XmlTest test = new XmlTest(testSuite);
         test.setName("all");
-        XmlPackage testPackage = new XmlPackage(options.getTestsPackage());
-        List<XmlPackage> testPackages = newArrayList(testPackage);
+        List<XmlPackage> testPackages = options.getTestsPackage().stream()
+                .map(XmlPackage::new)
+                .collect(toImmutableList());
```
Comment (findepi) [inline tempto-runner/src/main/java/io/prestosql/tempto/runner/TemptoRunner.java:126]: verify this is not an empty list?

File: tempto-runner/src/main/java/io/prestosql/tempto/runner/TemptoRunnerOptions.java [hunk 2]
```diff
@@ -105,9 +105,9 @@ public String getReportDir()
         return getValue(REPORT_DIR.getLongOpt()).get();
     }
 
-    public String getTestsPackage()
+    public Set<String> getTestsPackage()
```
Comment (findepi) [inline tempto-runner/src/main/java/io/prestosql/tempto/runner/TemptoRunnerOptions.java:108]: should be List, but i am ok

**Review by findepi — APPROVED** [review]
thanks!

File: tempto-runner/src/main/java/io/prestosql/tempto/runner/TemptoRunner.java (hunk 1 above)
Comment (kokosing) [reply tempto-runner/src/main/java/io/prestosql/tempto/runner/TemptoRunner.java:126]: It does not look like a mistake if test package is omitted from the point of view of testng.

**Pull request merged**